android/src/play/java/com/dooboolab/RNIap/RNIapModule.kt

```diff
@@ -108,7 +109,7 @@ class RNIapModule(
 
     @ReactMethod
     fun endConnection(promise: Promise) {
-        billingClient.endConnection()
+        billingClient?.endConnection()
         promise.safeResolve(true)
```

Comment:
I think it'd be better to handle when `billingClient` is null like below.


```
if (billingClient) {
  billingClient.endConnection()
  promise.safeResolve(true)

  return
}

promise.safeReject(false) // billingClient is null
```

How do you think?

Reply:
I've thought long and hard about this one. In the end, I think there are arguments for both ways of doing it. However, having to catch on a preventive `endConnection` call almost feels like punishing the developer that wanted to be extra careful. We can come back to it if needed